BLSQ/iaso
# IA-2996-IA-2949: Make Completeness table for PBF as a module as we can hide it and All search in french on modules

Explain what problem this PR is resolving
- Make Completeness table for PBF as a module as we can hide it
- Modules: text search only works with English module names
Related JIRA tickets : [IA-2996](https://bluesquare.atlassian.net/browse/IA-2996), [IA-2949](https://bluesquare.atlassian.net/browse/IA-2949)

## Self proofreading checklist


## Changes
- Add new module with **Completeness per Period** as name and link iaso_completeness with it
- Add **fr_name** key in modules to translate the module name and be able to search in french

## How to test
- Check on the modules page
- Remove the new module from the account
- Check if the iaso_completeness is still avaible in the permissions list
- Search modules with french name

## Print screen / video
- [Screencast from 2024-05-31 15-23-56.webm](https://github.com/BLSQ/iaso/assets/19631540/32538990-1231-4901-8e3a-dda0f783b5b6)
- [Screencast from 2024-05-31 15-24-27.webm](https://github.com/BLSQ/iaso/assets/19631540/73caf049-ff50-449e-b747-0922099655eb)



[IA-2996]: https://bluesquare.atlassian.net/browse/IA-2996?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IA-2949]: https://bluesquare.atlassian.net/browse/IA-2949?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
